TestData/TestPersonsPuData.cs

```diff
@@ -72,11 +72,11 @@ public static string[] PuDataList
+13270000200512312398200512312005123123981Testsson, Tor /Mats Erik/ Olof      Testvägen 10               12111Lillstad                            00000000000000000000                      201210222012102200000000                                                                                                                                                                                    00000000000000000000                                                                                                                                                                                                                                                                                                     Mats                                                                                                                    Testsson                                                                                                                          Testvägen 10                       12111Lillstad                                                                                                                                                                                                                                                                                                        000000000000  000000000000  _
+13270000992004391013191212129920043910133TOLVANSSON, Tolvan Ta /Tolv/        TOLFTE GATAN 12            11212TOLVSTAD                            00000000000000000000        038041        20140627        20140627                                                                                                                                                                                    00000000000000000000        299999  91299999  UTOMLÄNARE                    199016500  UTOMLÄNARE  HSN/STAB                              Ulp                 0000    UPPGIFT SAKNAS                9209999                                               8                                                               Tolvan                                                                                                                  Tolvansson                                                                                                                        TOLFTE GATAN 12                    11212TOLVSTAD                                                                                                                                                                                                                                                                                                        000000000000  000000000000  _
```

Comment:
Lowercase is good, but please lower case last name and preferably all other text too. The mixed casing looks even worse than the original upper case.